frontend/src/services/graphql/SagemakerNotebook/createSagemakerNotebook.js

```diff
@@ -1,6 +1,6 @@
 import { gql } from 'apollo-boost';
```

Comment:
I might be missing something, so please feel free correct me if I am wrong, but why we didn't put this code into modules? 
I thought we were going to localize the code as much as possible. So we could have `notebooks` folder and there will be the code only related to notebooks

Reply:
Yes, I also thought that the modules services will be inside the modules. Otherwise we end up with the same situation than at the beginning, customers having to go to multiple places to develop a single feature

Reply:
According to the planning (https://github.com/awslabs/aws-dataall/issues/398), the directory `src/services` is for the services that are used by several modules, and for any helpers related to the services.

What will happen (also according to the planning document) is that on each module that we update, we will move its related services inside that module -- By the end of this step, the only remaining services are the ones that are used in several modules.